src/Map/src/Twig/MapRuntime.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Twig;
+
+use Symfony\UX\Map\Map;
+use Symfony\UX\Map\Marker;
+use Symfony\UX\Map\Point;
+use Symfony\UX\Map\Renderer\RendererInterface;
+use Twig\Extension\RuntimeExtensionInterface;
+
+/**
+ * @author Simon André <smn.andre@gmail.com>
+ *
+ * @internal
+ */
+final class MapRuntime implements RuntimeExtensionInterface
+{
+    public function __construct(
+        private readonly RendererInterface $renderer,
+    ) {
+    }
+
+    /**
+     * @param array<string, mixed> $attributes
+     */
+    public function renderMap(
+        ?Map $map = null,
+        array $attributes = [],
+        ?array $center = null,
+        ?float $zoom = null,
+        ?array $markers = null,
+    ): string {
+        if (!$map instanceof Map) {
+            $map = new Map();
+        }
+        if (null !== $center) {
+            $map->center(Point::fromArray($center));
+        }
+        if (null !== $zoom) {
+            $map->zoom($zoom);
+        }
+        if (null !== $markers) {
+            foreach ($markers as $marker) {
+                $map->addMarker(Marker::fromArray($marker));
+            }
+        }
```

Comment:
We can make `$markers` not nullable, and simplify the code to:
```suggestion
        foreach ($markers as $marker) {
            $map->addMarker(Marker::fromArray($marker));
        }
```